packages/widget-v2/src/state/balances.ts

```diff
@@ -0,0 +1,23 @@
+import { BalanceRequest } from "@skip-go/client";
+import { atom } from "jotai";
+import { atomWithQuery } from "jotai-tanstack-query";
+import { skipClient } from "./skipClient";
+
+export const skipBalancesRequestAtom = atom<BalanceRequest | undefined>();
+
+export const skipBalancesAtom = atomWithQuery((get) => {
+  const skip = get(skipClient);
+  const params = get(skipBalancesRequestAtom);
+
+  return {
+    queryKey: ["skipBalances", params],
+    queryFn: async () => {
+      if (!params) {
+        throw new Error("No balance request provided");
+      }
+
+      return skip.balances(params);
+    },
+    retry: 1,
```

Comment:
can we add refetchInterval, maybe 30s or 1min or what ever you think it make sense

Reply:
I'll add 1 min for now, I'm a little worried about this endpoint xd

Reply:
maybe we should consider only refetching balances for a specific asset after you actually do a swap